docs/website/docs/index.md

```diff
@@ -159,7 +159,7 @@ runtime entirely or interface with custom accelerators.
 IREE supports the full set of these configurations using the same underlying
 technology.
 
-### Compiling models
+### :octicons-file-code-24: Compiling models
```

Comment:
Maybe this should get `:octicons-rocket-24:` :rocket: :wink:

Reply:
was going for consistency with these

https://github.com/iree-org/iree/blob/3ab9d4be1c491fdd455a91b255a288986550c786/docs/website/docs/guides/deployment-configurations/cpu.md?plain=1#L106 https://github.com/iree-org/iree/blob/3ab9d4be1c491fdd455a91b255a288986550c786/docs/website/docs/guides/deployment-configurations/gpu-cuda.md?plain=1#L88

(but with `24` instead of `16` on the homepage... I liked the 16px icons on subpages but 24px felt better here 🤷 https://primer.style/foundations/icons)

Reply:
and the previous section uses the rocket icon already

Reply:
Well this wasn't supposed to be a serious comment ;)